src/screens/Signup/styles.module.scss

```diff
@@ -0,0 +1,39 @@
+@import "scss/variables/_colors";
+
+.container {
+  background-color: $wild-sand;
+  border-top: 8px solid $wolox-blue;
+  margin-top: 51px;
+  max-width: 300px;
+  padding: 20px 0 24px;
+
+  img {
+    width: 212px;
+    margin-bottom: 16px;
+  }
+
+  .container-form {
+    padding: 0 23px 0 25px;
+
+    .form {
+      border-bottom: 2px solid $silver-chalice;
+      margin-bottom: 16px;
+      padding-bottom: 16px;
+
+      p {
+        color: $black;
+        font-size: 16px;
+        line-height: 24px;
+        margin-bottom: 8px;
+      }
+
+      input {
```

Comment:
same, and what about the error? also this input will be used as well on login, and is the same ui, what about creating a reusable input component that handles all the cases and also the error case.